doc/conf.py

```diff
@@ -56,6 +53,7 @@
     'matplotlib.sphinxext.plot_directive',
     'sphinxext.gen_rst',
     'sphinxext.directives',
+    'sphinxext.google_analytics'
```

Comment:
no, this it the file I have added manually. Otherwise we need to add https://pypi.python.org/pypi/sphinxcontrib-googleanalytics/ as a requirement